### PR TITLE
Remove inner border lines

### PR DESCRIPTION
**Pull request by kyi193**
I thought the grid looked stylistically unappealing to the eyes so I created custom ids for each cell if they were a one fo the border cells.

### CODE REVIEW

**Comment by kyi193** [comment]
Screen Shot 2020-08-19 at 8 20 41 PM

File: src/index.js [hunk 1]
```diff
@@ -76,20 +78,54 @@ function displayBoard() {
...
     tableBody.appendChild(row);
+    columnIndex += 1
   });
   table.appendChild(tableBody);
   document.querySelector('.game').appendChild(table)
   displayController();
 }
 
+const determineCellBorder = (rowIndex, columnIndex, rows, columns) => {
+  if (rowIndex === 0 && columnIndex === 0) {
```
Comment (squidapillar) [inline src/index.js:103]: topLeftcell is just top and left... maybe there's a better way to clean this up (same sorta thing goes on in the CSS classs names)

File: src/style.css [hunk 2]
```diff
@@ -22,6 +21,38 @@ td {
   border-radius: 50%;
 }
 
+#topLeftCell {
```
Comment (squidapillar) [inline src/style.css:24]: same - I think there's a better way to do this; somehow concatenate the two if it's top and left, etc.

File: src/index.js (hunk 1 above)
Comment (kyi193) [reply src/index.js:103]: How do I determine that in my function? If its a corner cell, taking out the special edge cases would result in missing spots. 
Screen Shot 2020-08-19 at 10 07 47 PM

Comment (kyi193) [reply src/index.js:103]: Hmm I got an idea I'll try out

**Pull request merged**